contracts/factorties/Factory.sol

```diff
@@ -0,0 +1,36 @@
+pragma solidity 0.8.9;
+
+import {Spigot} from "../modules/spigot/Spigot.sol";
+import {Escrow} from "../modules/escrow/Escrow.sol";
+
+contract Factory {
+    Spigot spigot;
+    Escrow escrow;
+    Spigot[] public SpigotArray;
+    Escrow[] public EscrowArray;
+    uint8 defaultRevenueSplit = 90;
+    uint32 defaultCRatio = 3000;
+
+    event DeployedSpigot(
+        address indexed spigotAddress
```

Comment:
Can you add all the params from funcs to the events too plz. 

We can only index 3 of the vars in an event so for DeploySpigot can you index  `owner` and `treasury` and for DeployEscrow can you index `minCRatio` and `borrower`